base/pkg/git.jl

```diff
@@ -105,20 +105,18 @@ function is_ancestor_of(a::AbstractString, b::AbstractString; dir="")
     readchomp(`merge-base $A $b`, dir=dir) == A
 end
 
-const GITHUB_REGEX =
-    r"^(?:git@|git://|https://(?:[\w\.\+\-]+@)?)github.com[:/](([^/].+)/(.+?))(?:\.git)?$"i
-
 function set_remote_url(url::AbstractString; remote::AbstractString="origin", dir="")
```

Comment:
First of all, no changes to `git.jl`, all modifications should go to `libgit2.jl`